assets/js/src/settings/utils.ts

```diff
@@ -0,0 +1,8 @@
+import MailPoet from 'mailpoet';
+import { ChangeEvent } from 'react';
+
+type Setter = (value: string) => any
+type Event = ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+export const onChange = (setter: Setter) => (e: Event) => setter(e.target.value);
+
+export const t = ([word]: TemplateStringsArray) => MailPoet.I18n.t(word);
```

Comment:
I used that to be able to use translated strings like
```
t`stringKey`
```
Instead of
```
t('stringKey')
```
Because it just feels like a string that way. That's just a syntactic-sugar, we can drop it if you think it doesn't worth it.

Reply:
I would prefer using a function. But maybe I'm just not used to it.